relay/channel/openai/buffered_stream.go

```
@@ -0,0 +1,223 @@
+	// Build choices for all indices
+	var choices []dto.OpenAITextResponseChoice
+	for idx := 0; idx < len(allIndices) || idx == 0; idx++ {
+		fr := finishReason[idx]
+		if fr == "" {
+			fr = constant.FinishReasonStop
+		}
+		choice := dto.OpenAITextResponseChoice{
+			Index:        idx,
+			FinishReason: fr,
+		}
+		choice.Message.Role = "assistant"
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "Message" from selector

(staticcheck)